frontend/src/views/Scan.vue

```diff
@@ -22,7 +22,7 @@ function onDecode (result) {
     <h2 class="pt-4">Scan QR/Barcode</h2>
     <div class="position-absolute d-flex flex-column">
       <div class="position-relative pe-3">
-        <router-link to="/workshops" class="btn float-end hover-darken mt-4 mb-4">
+        <router-link to="/products" class="btn float-end hover-darken mt-4 mb-4">
```

Comment:
Is het niet beter om hier naar de vorige pagina te navigeren? Want de scan pagina kan geopend worden op elke pagina en is verwarrend als je sluit en je gaat naar de product pagina